libminifi/test/resources/cn.ckey.pem

```diff
@@ -0,0 +1,31 @@
+Bag Attributes
```

Comment:
Might not be a bad idea, but not necessary.  At some point we can look into getting the Apache RAT running against our source and list as exclusions.  Actually, perhaps even just enumerating such resources on the wiki until we get a more formal check in place would be sufficient.  I had started the following https://cwiki.apache.org/confluence/display/MINIFI/Licensing+Information (and clearly forgot about updating it) a little while ago.  Maybe we can just highlight such files here after merge?